.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -1,5 +1,5 @@
-name: 🐞 Report a bug in Prefect 2.x
-description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+name: 🐛 Bug Report
+description: Report a bug or unexpected behavior in Prefect
```

Comment:
out of scope for this PR, but making a note that the sections and renderers here are overengineered IMO and result in lots of formatting bugs when people open issues